tools/vf-extensions-react/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 0.0.4
```

Comment:
I don't think you mean to update the `vf-extensions-react` changelog and package.json in this PR?

Reply:
No I did it on purpose. If I have to add changelog, I will have to update version as well, right? Otherwise, under what version to put changelog? Or we just update changelog with fictional future version?

Reply:
there's two things here:

- the `vf-extensions-react` changelog.md and package.json have been updated, but there aren't any changes to that component
- and then in `vf-button`, yes, no need to update the package.json lerna will update that. You're preparing the release notes with than anticipated new version, but we can update the version if needed when publishing — I think it will become clearer the first time you see the release process

Reply:
ok, I wont update files in `vf-extensions-react` (but your comment "could you update the changelog's for vf-extensions-react and vf-button" remains unresolved. was that a mistake?).

Reply:
Thanks. Have just resolved/approved.